app/institutions/dashboard/-components/institutional-users-list/component.ts

```diff
@@ -0,0 +1,63 @@
+import { action, computed } from '@ember-decorators/object';
+import { alias } from '@ember-decorators/object/computed';
+import { service } from '@ember-decorators/service';
+import Component from '@ember/component';
+import I18N from 'ember-i18n/services/i18n';
+import InstitutionModel from 'ember-osf-web/models/institution';
+
+export default class InstitutionalUsersList extends Component {
+    @alias('model.taskInstance.value') institution!: InstitutionModel;
```

Comment:
This should really be:
```suggestion
    @alias('model.taskInstance.value') institution?: InstitutionModel;
```
because the value is undefined until the task resolves.